env-defaults

```diff
@@ -184,7 +184,7 @@ WORK_FAILER_PERIOD_SEC=120
 # Used to avoid a tight loop
 WORK_ITEM_UPDATE_QUEUE_PROCESSOR_DELAY_AFTER_ERROR_SEC=1
 
-WORK_ITEM_UPDATE_QUEUE_PROCESSOR_COUNT=6
+WORK_ITEM_UPDATE_QUEUE_PROCESSOR_COUNT=3
```

Comment:
All of my successful testing in sandbox was with 3 processors, so changing the default.

Reply:
We use the env-defaults in deployed environments as well unless there is a Bamboo variable overriding it.